format/index.md

```diff
@@ -451,6 +452,7 @@ As discussed above, a JSON API document **MAY** be extended to include
 meta-information as `"meta"` members in several locations:
 
 * at the top-level
+* within the top level jsonapi object.
```

Comment:
Done. A lot of cleanup like this needs to happen document-wide. If you see any general cleanup issues, could you add them to #659?